.travis.yml

```
@@ -0,0 +1,12 @@
+language: python
+matrix:
+    include:
```

Plz use double space indentation for mappings
```suggestion
  include:
```